package.json

```diff
@@ -60,7 +60,7 @@
     "reflect-metadata": "^0.1.8",
     "rxjs": "5.4.0",
     "semver": "4.3.6",
-    "slickgrid": "github:anthonydresser/SlickGrid#2.3.23",
+    "slickgrid": "../SlickGrid",
```

Comment:
is this an intentional edit?

Reply:
Yeah, this change has changes in 3 repos. I'll update this as the other PR's go through